src/scripts.js

```diff
@@ -10,32 +10,54 @@ console.log('This is the JavaScript entry file - your code begins here.');
 import { getData } from './apiCalls';
 import UserRepository from './UserRepository';
 import User from './User';
+import HydrationRepo from './HydrationRepo';
 import Hydration from './Hydration';
 
-let userRepository, user;
-// function to get data from the API
+let userRepository, user, hydrationRepo, hydration, userIndex;
+
+// function to create random number for user
+const getRandomIndex = () => Math.floor(Math.random() * 50);
```

Comment:
Originally I thought this would be just a bonus but its really important to be able to see how the values are dynamic on page refresh